implementation/pom.xml

```diff
@@ -40,6 +40,12 @@
       <artifactId>microprofile-health-api</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.smallrye</groupId>
+      <artifactId>smallrye-health-extension-api</artifactId>
+      <version>${project.version}</version>
```

Comment:
I'd suggest this should be `provided`, and the regular Health API dependency above should be `provided` too, we generally do this in SmallRye projects to avoid dependency conflicts in target runtimes